java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/ui/AbstractLspHtmlViewer.java

```diff
@@ -121,11 +118,14 @@ private void load() {
                 notifyClose();
                 return;
             }
-            URL pageUrl = ctx.getPage();
-            Browser.Config c = initial.clone();
+            Browser.Config c = new Browser.Config();
```

Comment:
Is the `Config` used for anything? Can't it be deleted?

Reply:
Deleted.